examples/web-react/babel.config.js

```diff
@@ -1,6 +1,3 @@
 module.exports = {
-  presets: [
-    '@babel/preset-env',
-    '@babel/preset-react',
-  ],
+  presets: ['@babel/preset-env', '@babel/preset-react'],
```

Comment:
From 2 items up, I prefer the multiline array/object syntax (at least in `.js(x)`). It seems more readable to me, and you can also reorder items more easily.

What's your opinion on this? I'm afraid Prettier's decision is based just on line length…

Reply:
I think we should also add eslint for global scope. Not for just web-react package. That should solve this but I am not sure. I will give it a try in another PR.

Reply:
Well, I tried it in #93 but did not help.

Reply:
This is the problem with line width. You can [try it yourself](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBbCATArgGzgOjgA8AHCAJxgGcACAXhuAB0oabj4oMqkaBtFmzYByAAI5UYALRwsAeiowAnnhwBLWFMhQAZmoDmwgDSChYidNkLlqjTC3Q9+uRBIw10AIY5hpgLomUAC+ANwsIEYgru7QVMignuTkEADuAAqJCHEo3imeSnGRAEbknmAA1nAwAMqeqHAAMhpwyDreVHDFpRVV1SRlGvrIMORYnSBwqEVwGBgzDZ5Q+lie+nAAYhSonjDuS8ggnlgwEBEgABYwqDgA6udq8FT9YHDVWQ9qAG4PSgdgVIUQBoOpQ0qV9NtWu1xgArKhEaqDPAARSwEHgUJwHUi-XIIIORU80xwZxI5DsNzUGBg52QAA4AAw45IdG6lEgHMlwEGfFqRACOaPgYNc2UOVCkUDgMxmZ3IcEFanlYNWkKQbSx4w6qDUw1GWqRcFR6JaSBGY0iMCJlOptKQACZLaU1OolgBhCCoNUTKgAVjOWA6ABUidkNdiQJ8xgBJLgIGpgcluACCXGqNlN4bgQSCQA).

As we have 120 chars we will have always problem  with these configurations. But i think I should solve it with eslint. Maybe.

Reply:
I have solved this in #93 by using eslint for formatting javascript code (prettier must ignore it). But inside eslint configuration I am using prettier configuration for formatting. ESLint is not so strict in printWidth.